lib/components/app/app.css

```diff
@@ -31,15 +31,17 @@
   color: #ddd;
 }
 
-/* Don't show route viewer or trip plan link in the app menu on desktop as it is in the navbar */
-.app-menu-route-viewer-link,
-.app-menu-trip-planner-link {
+/* Don't show route viewer link in the app menu on desktop as it is in the navbar */
+.app-menu .app-menu-route-viewer-link,
+.app-menu .app-menu-trip-planner-link,
+.app-menu #app-menu-locale-selector {
```

Comment:
Why is the last one an id whereas the others are class names?

Reply:
Because the locale selector is generated whereas the others are fixed.